config/examples/Creality/CR-10 V3/Configuration.h

```diff
@@ -518,11 +518,11 @@
 //#define MAX31865_SENSOR_OHMS_1      100
 //#define MAX31865_CALIBRATION_OHMS_1 430
 
-#define TEMP_RESIDENCY_TIME          2  // (seconds) Time to wait for hotend to "settle" in M109
+#define TEMP_RESIDENCY_TIME          5  // (seconds) Time to wait for hotend to "settle" in M109
```

Comment:
2 sec was to low. Sometimes after starting moving, the machine was stoping again to re-heat